tests/fixtures/metadata_factory.py

```diff
@@ -0,0 +1,51 @@
+import factory
+from faker import Faker
+
+fake = Faker()
+
+
+def fake_file(type):
+    ext = 'pdf' if type.lower() == 'pdf' else None
+    domain = fake.domain_name()
+    file_path = fake.file_path(category=type.lower(), extension=ext)
+    return f"http://{domain}{file_path}"
+
+
+class TokenNFTMediaFactory(factory.DictFactory):
+    type = factory.Faker(
+        'random_element',
```

Comment:
I don't really like the tests being random. I believe they should be deterministic and test all scenarios. We had a situation when I was adding 'pdf' as new type that the test working locally, then worked in the PR CI and failed on `dev` branch, this is not good.

I know you are just using the code that we already had in the repository but I think we should at least have an issue to refactor this later.

Reply:
The factory boy dependency used to make these classes has a deterministic sequence generator, we could create a list of possible scenarios and each test would iterate through them.

Reply:
Shouldn't we do it in this PR? At least for the new tests that are being created

Or is this PR urgent?

Reply:
This change should also be made for the other tests that use random fixtures so I think that it belongs on it's own refactor PR.